frontend/packages/integration-tests-cypress/mocks/snapshot.ts

```diff
@@ -0,0 +1,82 @@
+export const testerDeployment = {
+  apiVersion: 'apps/v1',
```

Comment:
Hi, Can this be handled by the standard [Cypress Fixtures](https://docs.cypress.io/api/commands/fixture.html) api?

Reply:
Cypress fixtures are loaded asynchronously. Since these are used in multiple places(across various code blocks). It makes the design a bit complicated if we load it async.